web.go

```diff
@@ -221,3 +241,16 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	h.Target.ServeHTTP(w, r)
 }
+
+func (h *Handler) processResult(res result) string {
+	if !h.Jsonify {
+		return res.Message
+	}
+
+	result, err := json.Marshal(res)
+	if err != nil {
+		log.Printf("JSON() error, while encoding response: %v", err.Error())
```

Comment:
Some people may use remote log frameworks and not listen to standard error output.